cisco/asr1002-hx.yaml

```diff
@@ -0,0 +1,62 @@
+manufacturer: Cisco
+model: ASR1002-HX
+slug: asr1002-hx
+u_height: 2
+is_full_depth: false
```

Comment:
All devices with front-to-back airflow should be considered full-depth. (Installing a device in the same unit on the opposite face would inhibit airflow.)

Reply:
Set this to true.